curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-to-extract-values-from-objects.english.md

```diff
@@ -68,9 +68,9 @@ const tomorrow = HIGH_TEMPERATURES.tomorrow;
 
 // change code above this line
 
-console.log(yesterday) // should be not defined
-console.log(today); // should be 77
-console.log(tomorrow); // should be 80
+console.log(yesterday) // Should not be defined
```

Comment:
Same again

Reply:
I added a new test to be able to remove these comments.